openpype/hosts/flame/api/menu.py

```diff
@@ -26,6 +25,13 @@
 }
 
 
+def send_selection(selection):
```

Comment:
Why this is used in `isVisible` when it always return `True`?

Reply:
this is a hack-ish way of passing selection to singleton. I didn't find other way to do this.

Reply:
So you need to call that method before the tool is showed? In that case you should create wrapped callback for the action where it is called explicitly.

Reply:
Perhaps it is doable that way, but could you elaborate a little more on your idea?

Reply:
I believe this is solved now, thanks for suggestion ;)